tools/ImportExamples.ps1

```diff
@@ -90,10 +90,16 @@ function Get-Files {
                 $CommandValue = $null
                 if ($GraphProfile -eq "beta") {
                     $CommandValue = $BetaCommandGetVariantList[$Command]
+                    if(-not($CommandValue)){
+                        $CommandValue = $BetaCommandListVariantList[$Command]
+                    }
                         
                 }
                 else {
                     $CommandValue = $V1CommandGetVariantList[$Command]
+                    if(-not($CommandValue)){
+                        $CommandValue = $V1CommandListVariantList[$Command]
```

Comment:
What about other variants? Commands can have more than just `Get` and `List` variants:
``` powershell
➜ Find-MgGraphCommand -Command ".*-MgUser" | select Command, Method, URI, Variants

Command       Method URI              Variants
-------       ------ ---              --------
Get-MgUser    GET    /users/{user-id} {Get, GetViaIdentity}
Get-MgUser    GET    /users           {List}
New-MgUser    POST   /users           {Create, CreateExpanded}
Remove-MgUser DELETE /users/{user-id} {Delete, DeleteViaIdentity}
Update-MgUser PATCH  /users/{user-id} {Update, UpdateExpanded, UpdateViaIdentity, UpdateViaIdentityExpanded}
```
It is also worth noting that variant names for OData actions and functions are based on the command's verb. For example, `Revoke-MgUserSignInSession` will have a variant name of `Revoke` and `RevokeViaIdentity`.

Would it be better to use the HTTP method name to group the commands instead?